### PR TITLE
update samples metadata with new names and descriptions

### PR DESCRIPTION
**Pull request by christianvogt**
## Fixes 
https://issues.redhat.com/browse/DEVHAS-239

## Description

Updates the samples to override names and descriptions. Also filters the `.Net` sample out.

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

image

image

## How to test or reproduce?
View samples catalog.

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/utils/devfile-utils.ts [hunk 1]
```diff
@@ -16,18 +16,53 @@ export interface DevfileSample {
...
+const overrides: { [name: string]: SampleOverride } = {
+  'nodejs-basic': {
+    name: 'Node',
+    description: 'Node.js 16 application using Express 4.18.x',
+  },
+  'code-with-quarkus': {
+    name: 'Quarkus',
+    description: 'Java Quarkus application using Maven 4.0 and OpenJDK 11',
+  },
+  'java-springboo-basic': {
```
Comment (karthikjeeyar) [inline src/utils/devfile-utils.ts:34]:
```suggestion
  'java-springboot-basic': {
```

Comment (christianvogt) [reply src/utils/devfile-utils.ts]: fixed

File: integration-tests/support/pageObjects/createApplication-po.ts [hunk 2]
```diff
@@ -17,10 +17,10 @@ export const addComponentPagePO = {
 
 export const componentSamplesPagePO = {
   grid: '.catalog-page__grid',
-  nodejs: '[data-test="sample-Basic Node.js"]',
-  python: '[data-test="sample-Basic Python"]',
-  quarkus: '[data-test="sample-Basic Quarkus"]',
-  spring: '[data-test="sample-Basic Spring Boot"]',
+  nodejs: '[data-test="sample-Node.js"]',
```
Comment (karthikjeeyar) [inline integration-tests/support/pageObjects/createApplication-po.ts:20]:
```suggestion
  nodejs: '[data-test="sample-Node"]',
```

E2e test is failing

**Pull request merged**